src/test/java/com/hotels/mutantswarm/exec/SwarmResultsTest.java

```diff
@@ -172,4 +172,40 @@ public void testOutcomesListNull() {
     assertThat(outcomes, is(Collections.EMPTY_LIST));
   }
 
+  @Test
+  public void equalsNull() {
+    String suiteName = "testSuite";
+    SwarmResultsBuilder builder = new SwarmResultsBuilder(swarm, suiteName);
+    SwarmResults swarmResults = builder.build();
+    assertThat(swarmResults.equals(null), is(false));
+  }
+  
+  @Test
+  public void equalsSame() {
+    String suiteName = "testSuite";
+    SwarmResultsBuilder builder = new SwarmResultsBuilder(swarm, suiteName);
+    SwarmResults swarmResults = builder.build();
+    SwarmResults swarmResults2 = builder.build();
+    assertThat(swarmResults.equals(swarmResults2), is(true));
```

Comment:
```suggestion
    assertThat(swarmResults, is(swarmResults2));
```

Reply:
Here I'm also testing that the `equals()` method in SwarmResults works, so removing it would defeat its purpose right?

Reply:
`is()` probably calls `equals()` under the hood but you're right, since you're specifically testing the `equals()` method itself this is clearer.